src/css/in-page.css

```diff
@@ -210,6 +222,49 @@
   cursor: pointer;
   text-align: start;
   overflow: hidden;
+  position: relative;
+}
+
+.fx-relay-menu-masks-list button.is-loading::before {
+  position: absolute;
+  z-index: 0;
+  display: block;
+  width: 100%;
+  height: 100%;
+  background-color: var(--colorViolet10);
+  content: "";
+  left: 0;
+  top: 0;
+  animation-name: fade-in;
+  animation-duration: 2s;
+  transition-timing-function: linear;
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.fx-relay-menu-masks-list button.is-loading::after {
+  content: url(/icons/button-loading-dark.svg);
+  position: absolute;
+  z-index: 1;
```

Comment:
Could you add a comment explaining which element this z-index is intending to overlap? To help future contributors tell if it can be removed/upped.

Reply:
Comment added in 80402c57eaa99b1b1617691f015a18f1ed6fda02